CHANGELOG.md

```diff
@@ -160,7 +170,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - fix: only return previous payment id [`#6210`](https://github.com/opengovsg/FormSG/pull/6210)
 - build: merge release v6.45.0 to develop [`#6205`](https://github.com/opengovsg/FormSG/pull/6205)
 - build: release v6.45.0 [`#6200`](https://github.com/opengovsg/FormSG/pull/6200)
-- chore: bump version to v6.46.0 [`87fda0e`](https://github.com/opengovsg/FormSG/commit/87fda0e0ec4c937350ebd013a4a9762f4acee802)
+- chore: bump version to v6.46.0 [`767e7e5`](https://github.com/opengovsg/FormSG/commit/767e7e55c3e8f058963e890b1c249704d6db1459)
```

Comment:
I dunno why this is here... but also ... dun care! 😅

Reply:
hahaha hmm sus